controllers/users/create.go

```diff
@@ -35,7 +35,13 @@ func Create(au models.User, body []byte) (int, []byte) {
 		return 500, []byte("Error creating user")
 	}
 
-	u.Redact()
+	if u.MFA != nil && *u.MFA {
```

Comment:
this will cause a panic if u.MFA is nil, the dereference needs to happen inside the if call.